README.md

```diff
@@ -8,17 +8,25 @@ PokeReader is a 3gx plugin that allows viewing information about 3ds Pokemon gam
 
 This information can be used to RNG shiny and high IV Pokemon, similar to lua scripts on emulators.
 
+## Commands
+- X+Y to Lock/Unlock
+- D-pad keys to navigate when unlocked (right & left to get in and out of the different tabs)
+- Start + Select to pause game and enable manual frame advancing
+- Select to advance Main RNG frames once
+- Start or A to unpause
+- Start + Up (D-Pad) to Show/Hide the plugin
+
 ## Installing
 
 1. Update to the latest [Luma](https://github.com/LumaTeam/Luma3DS/releases) or set up [Citra](https://github.com/citra-emu/citra).
-1. Download the [latest PokeReader release](https://github.com/zaksabeast/PokeReader/releases/latest).
-1. Rename the `pokereader.3gx` file to `default.3gx`, then copy it to `/luma/plugins/` on your sd card (create the directory if it doesn't exist).
+2. Download the [latest PokeReader release](https://github.com/zaksabeast/PokeReader/releases/latest).
+3. Rename the `pokereader.3gx` file to `default.3gx`, then copy it to `/luma/plugins/` on your sd card (create the directory if it doesn't exist).
```

Comment:
Let's change this and the instruction in `Building` back to 1.

Starting with 1 renders a sequential list:
<img width="173" alt="Screenshot 2024-04-22 at 6 34 04 AM" src="https://github.com/zaksabeast/PokeReader/assets/19464008/e4e25a0a-216e-420e-99e4-472f66d49bac">

But if steps are ever added or re-arranged we don't need to renumber all of the steps for it to look clean in the file.